app/Migrations/Version20250212071939.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\Migrations\AbstractMigration;
+
+class Version20250212071939 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $table = $schema->createTable('mails_sent');
+
+        $table->addColumn('identifier', Types::GUID)->setLength(36)->setNotnull(true);
+        $table->addColumn('email', Types::STRING)->setLength(100)->setNotnull(true);
```

Comment:
Apparently the longest possible email address can be 320 characters. Might never happen off course but my gut feeling said 100 feels very arbitrary.